coding-concepts.js

```diff
@@ -15,45 +15,45 @@
 
 // --------------------1) What will this log?
 
-//const cohort = "LEARN 2022"
-//console.log(cohort.length)
+// const cohort = "LEARN 2022"
+//   console.log(cohort.length)
 
-// a) Your answer: This will log the amount of text in the constant.
-// b) Verify and explain: This logged 10 the amount of text in the constant.
+a) Your answer: This will log the amount of text in the string LEARN 2022.
+b) Verify and explain: This logged 10 the amount of text in the string because of the .length property.
 
 
 // --------------------2) What will this log?
 
-//const greeting = "Hello World!"
-//console.log(greeting[4])
+// const greeting = "Hello World!"
+//   console.log(greeting[4])
 
-// a) Your answer: This will log the letter o in Hello.
-// b) Verify and explain: This logged the letter o in Hello.
+a) Your answer: This will log the letter o in Hello.
+b) Verify and explain: This logged the letter o in Hello because the index [4] was called in the const greeting.
 
 
 // --------------------3) What will this log?
 
-//const languages = ["JavaScript", "Ruby", "Python", "C++"]
-//const index = 1
-// console.log(languages[index])
+// const languages = ["JavaScript", "Ruby", "Python", "C++"]
+// const index = 1
+//   console.log(languages[index])
 
-// a) Your answer: This will log Ruby.
-// b) Verify and explain: This logged Ruby.
+a) Your answer: This will log Ruby.
+b) Verify and explain: This logged Ruby because the const index was called and called the string located at index 1 of the array.
 
 
 // --------------------4) What will this log?
 
-const weekendDays = ["saturday", "sunday"]
- console.log(weekendDays.toUpperCase())
+// const weekendDays = ["saturday", "sunday"]
+//  console.log(weekendDays.toUpperCase())
 
-// a) Your answer: This will turn the text into uppercase.
-// b) Verify and explain: This .
+a) Your answer: This will turn the text into uppercase.
+b) Verify and explain: This logged a TypeError: weekendDays.toUpperCase is not a function.
```

Comment:
This TypeError occurs because .toUpperCase() only works on strings and not on arrays.  To make this work, you would need to access the individual values inside the array by bracket notation or iteration.